oaknational/Oak-Web-Application
# [PUPIL-759] - Cycle 1 support

Richard knows how to test the actually functionality, and has done the majority of that already.

**Devs**: Apologies this is a big one to review, it was a bit unavoidable with the way things evolved.

Due to it's size I think it's acceptable to scan for any obvious mistakes or anti-patterns, but I don't think it's required to pickup on any complex mistakes, as that would be very difficult in this instance. I think Richard has the functionality covered also.

## Description

This PR makes cycle 1 data available in the pupil browse experience. More info about the implementation, testing and AC can be found on the [ticket](https://www.notion.so/oaknationalacademy/Cycle-1-Support-Stage-1-4a044f919bda408782919481aab06fc5)

## How to test

1. Go to https://deploy-preview-2479--oak-web-application.netlify.thenational.academy/pupils/years

See [ticket](https://www.notion.so/oaknationalacademy/Cycle-1-Support-Stage-1-4a044f919bda408782919481aab06fc5) for details

## Checklist
